src/components/SQForm/SQFormButton.js

```diff
@@ -28,7 +30,9 @@ function SQFormButton({
       title={title}
       type={type}
       isDisabled={isSQFormButtonDisabled}
-      onClick={type === 'reset' ? handleReset : undefined}
+      onClick={
+        type === 'reset' ? handleReset : onClick ? handleClick : undefined
```

Comment:
I'm not the biggest fan of nested ternaries. This one isn't terrible. Would be possible do a `getClickHandler` to make determining the click handler for this button more readable? If others don't think this is necessary that's fine.

Reply:
Yeah, that's fair. I'll add a fix